src/main/java/com/avispl/symphony/dal/avdevices/power/gude/GudePDU8045Communicator.java

```diff
@@ -546,7 +555,7 @@ private void login() {
 	 * @param path url of the request
 	 * @return String full path of the device
 	 */
-	private String buildDeviceFullPath(String path) {
+	public String buildDeviceFullPath(String path) {
```

Comment:
Please change the method to private. If you using public please explain the reason

Reply:
I fixed it